dumb-jump.el

```diff
@@ -3,8 +3,8 @@
 ;; Author: jack angers and contributors
 ;; Url: https://github.com/jacktasia/dumb-jump
 ;; Version: 0.5.3
-;; Package-Requires: ((emacs "24.3") (s "1.11.0") (dash "2.9.0") (popup "0.5.3"))
-;; Keywords: programming
+;; Package-Requires: ((emacs "25.1") (s "1.11.0") (dash "2.9.0") (popup "0.5.3"))
```

Comment:
You don't have to update this, the code that relies on Emacs 25 is only conditionally evaluated, if xref is available.

Reply:
Ah, I see. Sorry! But what about `cl-defmethod`? Should I include cl-lib as requirement then?